### PR TITLE
homepage tweaks (how it works)

### LINKED ISSUE #472: Redesign: How It Works section (Home page)
Update the How It Works section on the home page.

1. Change `.how-it-works-title` `margin-bottom` to  70px
2. `.how-it-works` have `padding-top: 100px;` for screen widths 1024px and up
3. Reword the text under "How It Works" title so that the paragraph's last line doesn't have only 2 words. It looks very unbalanced right now.
4.  Remove `min-height` from `.how-it-works-tiles`
5. The sub titles “Learn” “Build” “Connect” needs `margin-top` 50px and` margin-bottom` 20px (Should probably add a class to the h3)
6. Mobile view of “How It Works” images to have `height` of 80px instead of 48px
7. At screen width 768 px, the `.tile` is too wide. Add media queries to make sure the `.tiles` width are balanced.

### Working on this task

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch

### PR DESCRIPTION
**Pull request by codyloyd**
fixes #472

### CODE REVIEW

**Comment by KevinMulhern** [comment]
👍 LGTM

File: app/views/static_pages/home/_how_it_works.html.erb [hunk 1]
```diff
@@ -2,27 +2,33 @@
...
-    <div class="tile">
-      <%= image_tag 'img-learn.svg', class: 'how-it-works-image', alt: 'how-it-works illustration' %>
-      <h3>Learn</h3>
-      <p>Learn from a curriculum with the best curated online tutorials, blogs, and courses.</p>
+    <div class="container">
+      <div class="tile">
+        <%= image_tag 'img-learn.svg', class: 'how-it-works-image', alt: 'how-it-works illustration' %>
+        <h3 class='subtitle'>Learn</h3>
+        <p>Learn from a curriculum with the best curated online tutorials, blogs, and courses.</p>
+    </div>
```
Comment (arun1595) [inline app/views/static_pages/home/_how_it_works.html.erb]: Indent it.

File: app/views/static_pages/home/_how_it_works.html.erb [hunk 2]
```diff
@@ -2,27 +2,33 @@
...
+      <p>This is the website we wish we had when we were learning on our own. We scour the internet looking for only the best resources to supplement your learning and present them in a logical order.</p>
     </div>
   </div> <!-- /.how-it-works-title -->
 
   <div class="how-it-works-tiles">
-    <div class="tile">
-      <%= image_tag 'img-learn.svg', class: 'how-it-works-image', alt: 'how-it-works illustration' %>
-      <h3>Learn</h3>
-      <p>Learn from a curriculum with the best curated online tutorials, blogs, and courses.</p>
+    <div class="container">
```
Comment (arun1595) [inline app/views/static_pages/home/_how_it_works.html.erb]: Shouldn't this be a `tile` class?

File: app/views/static_pages/home/_how_it_works.html.erb [hunk 3]
```diff
@@ -2,27 +2,33 @@
...
+      <div class="tile">
```
Comment (arun1595) [inline app/views/static_pages/home/_how_it_works.html.erb]: This should have a `container` class

Check the markup of the other two tiles.

File: app/views/static_pages/home/_how_it_works.html.erb (hunk 1 above)
Comment (codyloyd) [inline app/views/static_pages/home/_how_it_works.html.erb]: I think I'm going to componentize it.... good catch though.

File: app/views/static_pages/home/_how_it_works.html.erb (hunk 3 above)
Comment (codyloyd) [inline app/views/static_pages/home/_how_it_works.html.erb]: silly me.

**Comment by codyloyd** [comment]
@KevinMulhern @arun1595 I did the component thing... you should take another peek.

File: app/views/static_pages/home/_how_it_works.html.erb [hunk 4]
```diff
@@ -1,34 +1,31 @@
+ <% tiles = [
```
Comment (arun1595) [inline app/views/static_pages/home/_how_it_works.html.erb]: @codyloyd Can you move this to [ApplicationHelper](https://github.com/TheOdinProject/theodinproject/blob/redesign/app/helpers/application_helper.rb) and create an  instance variable in the `home` action of `StaticPagesController` to make it available to the views?

File: app/views/static_pages/home/_how_it_works_tile.html.erb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<div class="tile">
+    <%= image_tag tile[:image], class: 'how-it-works-image', alt: 'how-it-works illustration' %>
+    <h3 class='subtitle'><%= tile[:subtitle]%></h3>
+    <div class="description">
+      <p><%= tile[:description]%></p>
+    </div>
+</div>
```
Comment (arun1595) [inline app/views/static_pages/home/_how_it_works_tile.html.erb]: Nit. Add a line break.

File: app/views/static_pages/home/_how_it_works_tile.html.erb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<div class="tile">
+    <%= image_tag tile[:image], class: 'how-it-works-image', alt: 'how-it-works illustration' %>
```
Comment (arun1595) [inline app/views/static_pages/home/_how_it_works_tile.html.erb]: Nit. Indent it to 2 spaces.

Comment (arun1595) [inline app/views/static_pages/home/_how_it_works_tile.html.erb]: Don't we need the `img-responsive` class?

File: app/assets/stylesheets/components/how_it_works.scss [hunk 7]
```diff
@@ -21,20 +21,27 @@
...
+      }
       h3 {
         font-size: 1.1em;
         font-weight: 600;
-        margin-top: 1em;
-        margin-bottom: 1em;
       }
-      
+      .subtitle {
+        margin-top: 50px;
```
Comment (arun1595) [inline app/assets/stylesheets/components/how_it_works.scss]: Consider having a single property definition 

```
margin: 50px 0 20px;
```

File: app/views/static_pages/home/_how_it_works_tile.html.erb (hunk 5 above)
Comment (codyloyd) [inline app/views/static_pages/home/_how_it_works_tile.html.erb]: where?

File: app/views/static_pages/home/_how_it_works_tile.html.erb (hunk 6 above)
Comment (codyloyd) [inline app/views/static_pages/home/_how_it_works_tile.html.erb]: not in this case.

File: app/assets/stylesheets/components/how_it_works.scss (hunk 7 above)
Comment (codyloyd) [inline app/assets/stylesheets/components/how_it_works.scss]: nice catch

File: app/views/static_pages/home/_how_it_works_tile.html.erb (hunk 5 above)
Comment (arun1595) [inline app/views/static_pages/home/_how_it_works_tile.html.erb]: I meant the file is missing a newline at the end.

File: app/views/static_pages/home/_how_it_works.html.erb (hunk 4 above)
Comment (codyloyd) [inline app/views/static_pages/home/_how_it_works.html.erb]: I can do that.  It almost seems _tooooo_ abstracted, but doesn't hurt my feelings.  It certainly makes teh component cleaner

**Comment by codyloyd** [comment]
fixed.

**Pull request merged**